helpdesk_mgmt/controllers/main.py

```diff
@@ -47,13 +47,18 @@ def create_new_ticket(self, **kw):
         )
         email = http.request.env.user.email
         name = http.request.env.user.name
+        company = request.env.company
+        team_id_required_field = "helpdesk_mgmt_portal_team_id_required"
+        category_id_required_field = "helpdesk_mgmt_portal_category_id_required"
```

Comment:
This implementation is a little bit weird, why not getting the value directly in the dict below?